src/app/components/common/topbar/topbar.component.ts

```diff
@@ -313,7 +322,9 @@ export class TopbarComponent implements OnInit, OnDestroy {
   }
 
   onGoToLegacy() {
-    this.dialogService.confirm(T("Switch to Legacy User Interface?"), T(""), true, T("Continue")).subscribe((res) => {
+    this.dialogService.confirm(T("Warning:"), 
+      T("The Legacy User Interface is deprecated. The system should be managed from the new UI."), 
```

Comment:
Slight rewording.  Can we make the "new user interface" a link to the new user interface?
```suggestion
      T("The legacy user interface is deprecated. Please manage the system with the new user interface."), 
```

Reply:
Remember these items are all in the new UI. I think a link has been added to the warning in the old UI